test/unittest/StatisticsBackend/CMakeLists.txt

```
@@ -34,17 +34,24 @@ add_executable(statistics_backend_tests StatisticsBackendTests.cpp
     ${PROJECT_SOURCE_DIR}/src/cpp/topic_types/types.cxx
     ${PROJECT_SOURCE_DIR}/src/cpp/topic_types/typesPubSubTypes.cxx
     ${PROJECT_SOURCE_DIR}/src/cpp/types/EntityId.cpp)
+
+if(MSVC)
+    target_compile_definitions(statistics_backend_tests
+        PRIVATE _CRT_DECLARE_NONSTDC_NAMES=0 FASTDDS_STATISTICS_BACKEND_SOURCE)
+endif(MSVC)
+
 target_include_directories(statistics_backend_tests PRIVATE
     ${GTEST_INCLUDE_DIRS}
     ${PROJECT_SOURCE_DIR}/include
-    ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME}
+    ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME_UNDERSCORED}
     ${PROJECT_BINARY_DIR}/include
-    ${PROJECT_BINARY_DIR}/include/${PROJECT_NAME}
-    ${PROJECT_SOURCE_DIR}/src/cpp
-    ${PROJECT_SOURCE_DIR}/test/unittest/TestUtils)
+    ${PROJECT_BINARY_DIR}/include/${PROJECT_NAME_UNDERSCORED}
+    ${PROJECT_SOURCE_DIR}/src/cpp)
 
 target_link_libraries(statistics_backend_tests PUBLIC
-    ${GTEST_LIBRARIES} ${GMOCK_LIBRARIES} fastrtps)
+    ${GTEST_LIBRARIES} ${GMOCK_LIBRARIES} fastrtps fastcdr)
```

I don't think fastcdr is necessary, at least we are not using any CDR functionality. But maybe windows is picky and complaints because we are compiling code that would end up calling CDR if it was invoked? Please, check this.
```suggestion
    ${GTEST_LIBRARIES} ${GMOCK_LIBRARIES} fastrtps)
```